### DISCUSSION TITLE
macOS executable of Standalone Faster-Whisper

### DISCUSSION THREAD

**Discussion by Purfview** [Purfview/whisper-standalone-win#85] [Announcements]
Released Mac OS X executable of Standalone Faster-Whisper.

Should be compatible with Mac OS X v10.15 and above.

**Comment by Lycoan** [comment]
@Purfview, thank you for this release!

Could you consider creating an executable for Apple Silicon as well?
With similar faster-whisper implementations, such as whisper-ctranslate2, whisperx, I found transcribing approximately twice faster on this CPU type.

To improve performance, I tried to set `--compute_type=int8`, because I've read that it should be the best regarding speed.
However, no matter how I set is, 'int8' or 'auto", the selected type will be 'int8_float32' according to Verbose mode output:

`Supported compute types by CPU: {'float32', 'int8', 'int8_float32', 'int16'}`
`Selected compute type: int8_float32`

**Reply by Purfview** [reply]
> Apple Silicon

Atm, I don't have such.

**Reply by Lycoan** [reply]
I understand, thank you for work, anyway!

**Reply by tpfleging** [reply]
Amazing work by @Purfview  Seconded on hoping for a Apple Silicon executable.  Current MacOS executable works on Silicon but just takes an extra 15 seconds to start up on every use it seems
